addons/hr_holidays/models/hr_leave.py

```diff
@@ -664,8 +703,10 @@ def name_get(self):
                     target = leave.department_id.name
                 elif leave.holiday_type == 'category':
                     target = leave.category_id.name
-                else:
+                elif leave.employee_id:
                     target = leave.employee_id.name
+                else:
+                    target = ', '.join(leave.employee_ids.mapped('name'))
```

Comment:
What if there are more than 50 employees for example ? (eg: 24th of December at Odoo)